ansible/group_vars/all

```diff
@@ -48,7 +48,9 @@ whisk:
 #   runtimes_bypass_pull_for_local_images: optional, if true, allow images with a prefix that matches
 #       {{ runtimes_local_image_prefix }} to skip docker pull in invoker even if the image is not part of the blackbox set
 #
-runtimesManifest: "{{ runtimes_manifest | default(lookup('file', openwhisk_home ~ '/ansible/files/runtimes.json') | from_json) }}"
+
+manifestfile: "{{ manifest_file | default('/ansible/files/runtimes-nodeonly.json') }}"
```

Comment:
I would not change the default, leave the default as runtimes.json with all the runtimes, and then for places in travis that we only need nodejs then use ansible `-e manifestfile=<path/to/>runtimes-nodeonly.json`
Also don't rename `runtimes.json` for now keep it, as other repos might be replacing this file before they start the deploy as a way to override the default today.